lib/px/block/templates/block_template.mustache

```diff
@@ -3,172 +3,38 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Access to this page has been denied.</title>
-    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300" rel="stylesheet">
-    <style>
-        html, body {
-            margin: 0;
-            padding: 0;
-            font-family: 'Open Sans', sans-serif;
-            color: #000;
-        }
-
-        a {
-            color: #c5c5c5;
-            text-decoration: none;
-        }
-
-        .container {
-            align-items: center;
-            display: flex;
-            flex: 1;
-            justify-content: space-between;
-            flex-direction: column;
-            height: 100%;
-        }
-
-        .container > div {
-            width: 100%;
-            display: flex;
-            justify-content: center;
-        }
-
-        .container > div > div {
-            display: flex;
-            width: 80%;
-        }
-
-        .customer-logo-wrapper {
-            padding-top: 2rem;
-            flex-grow: 0;
-            background-color: #fff;
-            visibility: {{logoVisibility}};
-        }
-
-        .customer-logo {
-            border-bottom: 1px solid #000;
-        }
-
-        .customer-logo > img {
-            padding-bottom: 1rem;
-            max-height: 50px;
-            max-width: 100%;
-        }
-
-        .page-title-wrapper {
-            flex-grow: 2;
-        }
-
-        .page-title {
-            flex-direction: column-reverse;
-        }
-
-        .content-wrapper {
-            flex-grow: 5;
-        }
-
-        .content {
-            flex-direction: column;
-        }
-
-        .page-footer-wrapper {
-            align-items: center;
-            flex-grow: 0.2;
-            background-color: #000;
-            color: #c5c5c5;
-            font-size: 70%;
-        }
-
-        @media (min-width: 768px) {
-            html, body {
-                height: 100%;
-            }
-        }
-    </style>
-    <!-- Custom CSS -->
-    {{#cssRef}}
-        <link rel="stylesheet" type="text/css" href="{{cssRef}}"/>
-    {{/cssRef}}
+    <meta name="description" content="px-captcha">
+    <title>Access to this page has been denied</title>
+    {{{cssRef}}}
```

Comment:
Do these generate properly in the template? They need the `link` and `script` tags in the HTML if they exist, and if they are empty then these elements shouldn't appear at all in the HTML. Typically for mustache templates, we do:

```
    {{#cssRef}}
        <link rel="stylesheet" type="text/css" href="{{{cssRef}}}">
    {{/cssRef}}
```

and 

```
    {{#jsRef}}
        <script src="{{{jsRef}}}"></script>
    {{/jsRef}}
```

Reply:
Thanks